### PR TITLE
release/v0.22.1

### PR DESCRIPTION
**Pull request by carpawell**
Update changelog for `v0.22.1` release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,24 @@ Changelog for NeoFS Node
...
 
+## [0.21.1] - 2021-07-07
+
+### Added
+- `GetCandidates` method to morph client wrapper ([#647](https://github.com/nspcc-dev/neofs-node/pull/647)).
+- All-in-One Docker image that contains all NeoFS related binaries ([#662](https://github.com/nspcc-dev/neofs-node/pull/662)).
+- `--version` flag to Storage Node binary. ([#664](https://github.com/nspcc-dev/neofs-node/issues/664)).
+
+### Changed
+- Do not use multisignature for audit operations ([#658](https://github.com/nspcc-dev/neofs-node/pull/658)).
```
Comment (alexvanin) [inline CHANGELOG.md:14]: It's rather bug fix than change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -453,6 +471,7 @@ NeoFS-API v2.0 support and updated brand-new storage node application.
 First public review release.
 
 [Unreleased]: https://github.com/nspcc-dev/neofs-node/compare/v0.22.0...master
```
Comment (cthulhu-rider) [inline CHANGELOG.md:473]:
```suggestion
[Unreleased]: https://github.com/nspcc-dev/neofs-node/compare/v0.22.1...master
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,24 @@ Changelog for NeoFS Node
...
+### Changed
+- Do not use multisignature for audit operations ([#658](https://github.com/nspcc-dev/neofs-node/pull/658)).
+- Do not check NeoFS version in `LocalNodeInfo` requests and `Put` container operations. `v2.7.0` is genesis version of NeoFS. ([#660](https://github.com/nspcc-dev/neofs-node/pull/660)).
+- All error calls of CLI return `1` exit code ([#657](https://github.com/nspcc-dev/neofs-node/issues/657)).
+
+### Fixed
+- Skip audit for containers without Storage Groups ([#659](https://github.com/nspcc-dev/neofs-node/issues/659)).
+
+### Updated
+- NeoFS API Go: [v1.28.1](https://github.com/nspcc-dev/neofs-api-go/releases/tag/v1.28.1).
```
Comment (alexvanin) [inline CHANGELOG.md:22]: Update to v1.28.2 in this PR.

File: CHANGELOG.md (hunk 1 above)
Comment (carpawell) [reply CHANGELOG.md:14]: moved to fix section

File: CHANGELOG.md (hunk 3 above)
Comment (carpawell) [reply CHANGELOG.md:22]: updated

File: CHANGELOG.md (hunk 2 above)
Comment (carpawell) [reply CHANGELOG.md:473]: fixed

File: go.sum [hunk 4]
```diff
@@ -372,6 +372,8 @@ github.com/nspcc-dev/neofs-api-go v1.26.1/go.mod h1:SHuH1Ba3U/h3j+8HHbb3Cns1LfMl
 github.com/nspcc-dev/neofs-api-go v1.27.1/go.mod h1:i0Cwgvcu9A4M4e58pydbXFisUhSxpfljmuWFPIp2btE=
 github.com/nspcc-dev/neofs-api-go v1.28.1 h1:u+JbrisfLCXICamU3ie+UNPLwpjFdf14P2M+yEKnblw=
 github.com/nspcc-dev/neofs-api-go v1.28.1/go.mod h1:YRIzUqBj/lGbmFm8mmCh54ZOzcJKkEIhv2s7ZvSLv3M=
```
Comment (alexvanin) [inline go.sum:374]: go mod tidy

Comment (carpawell) [reply go.sum]: tidied

**Review by alexvanin — APPROVED** [review]

**Pull request merged**